### PR TITLE
output test exit code

### PR DESCRIPTION
**Pull request by charlierudolph**
@kevgo

### CODE REVIEW

File: features/test-exit-status.feature [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    Given Tertestrial is running inside the "no-output-tests" example application
+
+
+  Scenario: with a test that succeeds
+    When sending the command:
+      """
+      {"filename": "foo"}
+      """
+    Then I see "run-test-success"
+    And I see "Exit Code: 0"
```
Comment (kevgo) [inline features/test-exit-status.feature]: keep this lowercase? Its not a title or proper noun.

```cucumber
And I see "exit code: 0"

File: example-applications/no-output-tests/tertestrial.yml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+actions:
+
+  - match:
+      filename: 'foo'
```
Comment (kevgo) [inline example-applications/no-output-tests/tertestrial.yml]: filename: 'success'

File: example-applications/no-output-tests/tertestrial.yml [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+    command: 'bin/run-test-success'
+
+  - match:
+      filename: 'bar'
```
Comment (kevgo) [inline example-applications/no-output-tests/tertestrial.yml]: filename: 'failure'

File: features/test-exit-status.feature [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+  Background:
+    Given Tertestrial is running inside the "no-output-tests" example application
+
+
+  Scenario: with a test that succeeds
+    When sending the command:
+      """
+      {"filename": "foo"}
+      """
+    Then I see "run-test-success"
```
Comment (kevgo) [inline features/test-exit-status.feature]: I think this line is not really necessary here. It doesn't add documentation. Remove it?

**Comment by kevgo** [comment]
Looks great! 🚢 after the comments have been addressed

File: example-applications/no-output-tests/tertestrial.yml (hunk 2 above)
Comment (charlierudolph) [inline example-applications/no-output-tests/tertestrial.yml]: haha had that at first. Then followed the long-running-tests example which used an empty set and `filename: foo`

File: features/test-exit-status.feature (hunk 1 above)
Comment (charlierudolph) [inline features/test-exit-status.feature]: fixed

File: example-applications/no-output-tests/tertestrial.yml (hunk 2 above)
Comment (charlierudolph) [inline example-applications/no-output-tests/tertestrial.yml]: fixed

File: example-applications/no-output-tests/tertestrial.yml (hunk 3 above)
Comment (charlierudolph) [inline example-applications/no-output-tests/tertestrial.yml]: fixed

File: features/test-exit-status.feature (hunk 4 above)
Comment (charlierudolph) [inline features/test-exit-status.feature]: removed

**Comment by kevgo** [comment]
Looks great now!  🚢

**Pull request closed without merging**